README.md

```diff
@@ -72,6 +72,11 @@ Some possible tasks to add-to/replace the default options:
 - setup a reverse proxy for remote shell (outbond from container host to remote support system)
 - create a container project from defined template
 
+
```

Comment:
Very good/astute addition of documentation.  Developers have a habit of being very fast and Agile - but not always so good at documenting the new features before we race on to the next feature task.  ;)